src/executor/evaluate/mod.rs

```diff
@@ -408,11 +408,42 @@ async fn evaluate_function<'a, T: 'static + Debug>(
         Function::Pi() => {
             { Ok(Evaluated::from(Value::F64(std::f64::consts::PI))) }.map(Evaluated::from)
         }
-        Function::Trim(expr) => match eval_to_str(expr).await? {
-            Nullable::Value(string) => Ok(Value::Str(string.trim().to_owned())),
-            Nullable::Null => Ok(Value::Null),
+        Function::Trim { expr, trim_where } => {
+            let expr_str = match eval_to_str(expr).await? {
+                Nullable::Value(str) => str,
+                Nullable::Null => return Ok(Value::Null).map(Evaluated::from),
+            };
+            let expr_str = expr_str.as_str();
+
+            if expr_str.is_empty() {
+                return Ok(Value::Str(expr_str.to_owned())).map(Evaluated::from);
+            }
```

Comment:
Result of `TRIM(NULL FROM "")` is expected to be `NULL` but, it returns empty string.
This looks not necessary to have.

Reply:
Fixed it